hudi-client/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -172,6 +182,15 @@ public void updateFinalizeWriteMetrics(long durationInMs, long numFilesFinalized
     }
   }
 
+  public void updateIndexMetrics(final String action,final long durationInMs) {
```

Comment:
nit: single space after comma and next argument?  `action, final long` ? (I am surprised checkstyle does not have this. hmm.. @bvaradar )